pkgs/development/python-modules/spacy/default.nix

```
@@ -34,7 +34,9 @@ buildPythonPackage rec {
 
   prePatch = ''
     substituteInPlace setup.py \
-      --replace "plac<1.0.0,>=0.9.6" "plac>=0.9.6"
+      --replace "plac<1.0.0,>=0.9.6" "plac>=0.9.6" \
+      --replace "thinc>=7.0.8,<7.1.0" "thinc>=7.0.8" \
```

kind of a small point, but if you want to get rid of version bounds, you can just name the package itself
```suggestion
      --replace "thinc>=7.0.8,<7.1.0" "thinc" \
```